pkg/dial/dialer.go

```diff
@@ -18,20 +19,22 @@ import (
 	"storj.io/storj/pkg/transport"
 )
 
-// Dialer is a kademlia dialer
+var mon = monkit.Package()
+
+// Dialer is a dialer
```

Comment:
😂love that, can we maybe be a little more descriptive? Not your fault, this is a common occurrence across the codebase.